docs/src/components/pricing/PricingTable.tsx

```diff
@@ -766,7 +766,7 @@ export default function PricingTable({
             bgcolor: (theme) =>
               theme.palette.mode === 'dark'
                 ? alpha(theme.palette.primaryDark[900], 0.3)
-                : alpha(theme.palette.grey[50], 0.4),
+                : alpha(theme.palette.grey[50], 0),
```

Comment:
@mbrookes 

> Table rows have a hover indicator but clicking anything other than the Data grid accordion doesn't do anything, breaking expectations.

Iterated with removing the background hover of the rows but I personally enjoy the light bg on hover, it's an even stronger helper to not lose in which row I'm at. To me, what ultimately gives an expectation of the ability to interact is the cursor pointer. So when changing from the Data Grid row to another one, the cursor changing makes it clear that the other ones are not as interactive. It doesn't feel like a problem to me. But again, it seems like a matter of what feels right. So, more opinions are nice: @siriwatknp and @oliviertassinari what about you, what do you think?

Reply:
I think the light bg helps when I try to compare between each plan (give a line to my eye, better than not having). I vote for keeping it and remove the bg only in touch device.